README.md

```diff
@@ -3,6 +3,8 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/elastic/cloudbeat)](https://goreportcard.com/report/github.com/elastic/cloudbeat)
 [![Build Status](https://internal-ci.elastic.co/buildStatus/icon?job=cloudbeat%2Fcloudbeat-mbp%2Fmain)](https://internal-ci.elastic.co/job/cloudbeat/job/cloudbeat-mbp/job/main/)
 
+The beta software program product licensed hereunder is still in its testing phase and is provided on an "as is" and "as available" basis
```

Comment:
As you suggested let's use what metricbeat did - https://github.com/elastic/beats/tree/fabdcc40014dd3aa48dbf8951e989e68fcabb9e9/metricbeat#metricbeat
WARNING: Metricbeat is currently still in an experimental phase and under heavy development.